_docs/concepts/traffic-management/overview.md

```diff
@@ -1,44 +1,36 @@
 ---
 title: Overview
-overview: Provides a conceptual overview of traffic management principles in Istio and the kind of features enabled by these principles.
+overview: Provides a conceptual overview of traffic management in Istio and the features enabled by its traffic management principles.
                 
 order: 0
 
 layout: docs
 type: markdown
 ---
 
-Existing container orchestration platforms like Kubernetes, and Mesos, and
-other microservice frameworks allow operators to control when a particular
-set of pods/VMs should receive traffic (e.g., by adding/removing specific
-labels). Unlike existing techniques, Istio decouples traffic flow and infrastructure
-scaling. 
+This page provides an overview of how traffic management works in Istio, including the benefits of its traffic management principles. It assumes that you've already read [What Is Istio?]({{home}}/docs/concepts/what-is-istio/overview.html) and are familiar with Istio's high-level architecture. You can find out more about individual traffic management services and features in the other guides in this section.
+
+## Istio-Manager and Envoy
+
+The core component used for traffic management in Istio is the [Istio-Manager]({{home}}/docs/concepts/traffic-management/manager.html) service, which manages and configures all the Envoy proxy instances deployed in a particular Istio service mesh. It lets you specify what rules you want to use to route traffic between Envoy proxies and configure failure recovery features such as timeouts. It also maintains a canonical model of all the services in the mesh and uses this to let Envoys know about the other instances in the mesh via its discovery service.
```

Comment:
is the Istio-Manager -> is Istio-Manager

The link to Istio-Manager can be relative.